### PR TITLE
Feature/logging and kryptnostic services

### PR DESCRIPTION
**Pull request by juliannaelamb**
Added tracking for document sizes.

### CODE REVIEW

**Comment by juliannaelamb** [comment]
@nickdhewitt @makosblade @geekbeast

File: src/main/java/com/kryptnostic/api/v1/client/DefaultKryptnosticClient.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public DefaultKryptnosticClient( KryptnosticServicesFactory factory, Kryptnostic
                 context,
                 factory.createDocumentApi(),
                 factory.createMetadataApi(),
-                factory.createSharingApi() );
+                factory.createSharingApi());
```
Comment (geekbeast) [inline src/main/java/com/kryptnostic/api/v1/client/DefaultKryptnosticClient.java]: Do you have our auto-formatting preferences installed?

File: src/main/java/com/kryptnostic/api/v1/storage/DefaultStorageClient.java [hunk 2]
```diff
@@ -379,7 +382,8 @@ public String appendObject( ObjectMetadata objectMetadata, String body ) throws
         EncryptedSearchSharingKey encryptedSearchSharingKey = privKey.calculateSharingKey( bridgeKey );
         List<MetadataRequest> mreq = prepareMetadata( metadata, encryptedSearchSharingKey );
         uploadMetadata( mreq );
-        return objectApi.appendObject( objectMetadata.getId(), blockToAppend ).getData();
+        AppendObjectRequest req = new AppendObjectRequest( body.length(), blockToAppend );
+        return objectApi.appendObject( objectMetadata.getId(), req ).getData();
```
Comment (geekbeast) [inline src/main/java/com/kryptnostic/api/v1/storage/DefaultStorageClient.java]: There's no need to create a new AppendObjectRequest. 

Server side you can append the length of the body to the existing length of the metadata and update it.

**Comment by geekbeast** [comment]
No client side changes required for now.

**Pull request closed without merging**